### ISSUE TITLE
Pre-populate phone number field for ANC woman, if applicable

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-chw-anc#62]
In the [ANC data dictionary](https://docs.google.com/spreadsheets/d/1hZrD3P8X3YAd-JvlXdT4TGKWJ87NlaLKjJlPrJZFwMY/edit#gid=0), ANC Registration tab, row 10, please see the comment about trying to pre-populate the woman's phone number with the phone number provided in her family member registration form, if applicable.

------------------------------------------------------------------------------------------------------
BA: No difference.

**Comment by msschroeder** [comment]
QA 6 June Build:
In the [ANC data dictionary (BA)](https://docs.google.com/spreadsheets/d/1feKFVFtCZsGVzwEglbDZ4f5uP84F8wL2jwwXvfqUgP0/edit?usp=sharing), CHW 1 - ANC Registration, row 18, BA has the same comment. I have highlighted in pink.

I did notice this condition did apply when the woman was the head of household, this number was populated. However, if she was not the head of household, this number remained blank.
